### PR TITLE
Update changelog for azure-spring-boot-bom:3.6.1

### PR DESCRIPTION
**Pull request by chenrujun**
Update changelog for azure-spring-boot-bom:3.6.1

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , please help to review this PR.

File: sdk/boms/azure-spring-boot-bom/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 3.6.1 (2021-07-12)
+This release is compatible with Spring Boot 2.5.
+### New Features
+- Upgrade to [spring-boot-dependencies:2.5.2](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.2/spring-boot-dependencies-2.5.2.pom).
+- Upgrade to [spring-cloud-dependencies:2020.0.3](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.3/spring-cloud-dependencies-2020.0.3.pom).
```
Comment (saragluna) [inline sdk/boms/azure-spring-boot-bom/CHANGELOG.md:7]: This is not related to the spring boot projects.

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/boms/azure-spring-boot-bom/CHANGELOG.md (hunk 1 above)
Comment (chenrujun) [reply sdk/boms/azure-spring-boot-bom/CHANGELOG.md]: Deleted.

**Review by saragluna — APPROVED** [review]
LGTM.
